configs/jest-config-spirit/jsdom/setup/setupAfterEnv.js

```diff
@@ -1,6 +1,14 @@
 import '@testing-library/jest-dom';
+import { createRequire } from 'module';
 import { TextEncoder, TextDecoder } from 'util';
 import ResizeObserverPolyfill from 'resize-observer-polyfill';
+import { registerA11yMatchers } from '../matchers/index.js';
```

Comment:
```suggestion
import { registerA11yMatchers } from '../matchers';
```
? Maybe this is enough?